code/webauthn/frontend/src/pages/index.tsx

```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import { Layout } from '../components/Layout';
+import React from 'react';
+
+export const AA_FACTORY_ADDRESS = '0x<YOUR_AA_FACTORY_ADDRESS>';
```

Comment:
It looks like you are already putting this into environment variables, and some Set step code replacement that I'm not familiar with. What's preventing reading these on build as the NEXT_ prefixed variables instead of this?

Reply:
I could do that here as well